packages/cli/src/util/dev/router.ts

```diff
@@ -56,7 +57,8 @@ export async function devRouter(
   phase?: HandleValue | null
 ): Promise<RouteResult> {
   let result: RouteResult | undefined;
-  let { query, pathname: reqPathname = '/' } = url.parse(reqUrl, true);
+  let { pathname: reqPathname = '/', search: reqSearch } = url.parse(reqUrl);
```

Comment:
With `url.parse()` having been deprecated, perhaps we should switch to the new, futureproof method:

```javascript
let { pathname: reqPathname = '/', search: reqSearch } = new URL(reqUrl);
```

Note that the URL must have a protocol and domain, otherwise the call would look like `new URL(reqUrl, 'http://localhost')`.

Reply:
Its not really deprecated https://nodejs.org/api/url.html#urlparseurlstring-parsequerystring-slashesdenotehost

<img width="945" alt="image" src="https://user-images.githubusercontent.com/229881/183964044-f9fe30ce-35d1-4a8c-b297-d99ff3dae99e.png">

Reply:
Ha, revoked!